packages/config/config/devices/0x0234/zdb5100.json

```diff
@@ -17,17 +17,50 @@
 	},
 	"supportsZWavePlus": true,
 	"paramInformation": {
-		"1": {
-			"label": "Operating pushbutton(s) for dimmer",
-			"description": "This parameter specifies which pushbutton(s) that controls the built-in dimmer.",
+		"1[0x01]": {
+			"label": "Pusbutton 1 controls the dimmer",
+			"description": "Enabling this parameter, lets pushbutton 1 control the built-in dimmer.",
```

Comment:
I think the comma behind `parameter` does not belong there. (same for the other params)